types/repo/SignedMessageRepo/SignedPostNotificationData.ts

```diff
@@ -91,6 +91,10 @@ export interface PostMaker {
 export interface PostMakerData {
   profile_pic_url?: string;
   username: string;
+  emoji_name: string | null;
+  emoji_code: string | null;
+  color_name: string | null;
+  color_code: string | null;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new fields `emoji_name`, `emoji_code`, `color_name`, and `color_code` have been added to the `PostMakerData` interface. Ensure that these fields are properly validated before use, as they could be potential sources of security vulnerabilities if not handled correctly. For instance, if these values are directly inserted into HTML or CSS without proper sanitization, it could lead to Cross-Site Scripting (XSS) attacks.

```diff
 export interface PostMakerData {
   profile_pic_url?: string;
   username: string;
+  emoji_name: string | null;
+  emoji_code: string | null;
+  color_name: string | null;
+  color_code: string | null;
 }
```
Also, consider adding comments to describe what these new fields represent and how they should be used. This will improve code readability and maintainability.


<!-- This is an auto-generated comment by OSS CodeRabbit -->